Remotion/Web/Core/UI/SmartPage.cs

```diff
@@ -543,6 +624,40 @@ public bool? EnableSmartFocusing
     set { _enableSmartFocusing = value; }
   }
 
+  /// <summary>
+  /// Gets or sets the flag that determines whether a Content-Security-Policy header is set.
+  /// The value of the header is determined by <see cref="GetCspHeader"/>.
+  /// </summary>
+  [Description("The flag that determines whether a Content-Security-Policy header is set.")]
+  [Category("Behavior")]
+  [DefaultValue(false)]
+  public bool IsCspEnabled
```

Comment:
Please move the new properties after the smart-focusing block. right now it's in the middle of a group of related members.